plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -292,4 +298,36 @@ private URL getIncrementalsMirror() {
         }
         return jenkinsIncrementalsRepo;
     }
+
+
+    /**
+     * Prints out the Plugin Management Tool version
+     */
+    public void showVersion() {
+        try (InputStream propertyInputStream = getPropertiesInputStream("/.properties")) {
+            if (propertyInputStream == null) {
+                System.out.println("No version information available");
+                return;
+            }
+            final Properties properties = new Properties();
+            properties.load(propertyInputStream);
+            System.out.println("Version: " + properties.getProperty("project.version"));
+        } catch (IOException e) {
+            System.out.println("No version information available");
```

Comment:
This isn't right as no error is thrown.
There's probably 2 ways to handle it, rethrow the exception and handle exitting higher up, or use a junit rule like this one here:
http://stefanbirkner.github.com/system-rules/#ExpectedSystemExit

Reply:
So I did try to notify the user that there was an error, but you're saying that I actually need to throw the exception?

Reply:
I would, this shouldn't ever happen and it shows that this is a broken build and probably shouldn't be used?

Reply:
That's a fair test